src/FOMObot/App.hs

```diff
@@ -3,21 +3,33 @@ module FOMObot.App
     ) where
 
 import System.Environment (getEnv)
-import Control.Monad (void, when)
+import Control.Lens (uses, views, (^.))
+import Control.Monad (void, when, unless)
 import qualified Web.Slack as Slack
 
 import FOMObot.Helpers.Bot
+import FOMObot.Helpers.Redis
 import FOMObot.Types.AppState
 import FOMObot.Types.Bot
 import FOMObot.Types.BotConfig
 import FOMObot.Types.BotState
 
 runApp :: Slack.Event -> Bot ()
-runApp m@(Slack.Message cid (Slack.UserComment _) _ _ _ _) = do
+runApp m@(Slack.Message cid (Slack.UserComment uid) _ _ _ _) = do
     ignoreFOMOChannel <- isFOMOChannel cid
-    if ignoreFOMOChannel
-       then return ()
-       else (`when` alertFOMOChannel cid) =<< processMessage m
+    unless ignoreFOMOChannel $ do
```

Comment:
I'm not sure if these operations are expensive, but thoughts on...

```haskell
isFOMO <- isFOMOChannel cid
isDM <- isDMChannel uid cid

case (isFOMO, isDM) of
    (True, _) -> return ()
    (_, True) -> processCommand m
    (_, False) -> (`when` alertFOMOChannel cid) =<< processMessage m
```

For me personally, that's easier to understand the different branches of code since it feels like it's laid out on one level. Thoughts?